src/python/WMCore/MicroService/Unified/MSCore.py

```diff
@@ -0,0 +1,52 @@
+"""
+File       : MSCore.py
+Author     : Valentin Kuznetsov <vkuznet AT gmail dot com>
+Description: MSCore class provides core functionality of the MS.
+"""
+# futures
+from __future__ import division, print_function
+
+# WMCore modules
+from WMCore.MicroService.Unified.Common import getMSLogger
+from WMCore.Services.ReqMgr.ReqMgr import ReqMgr
+from WMCore.Services.ReqMgrAux.ReqMgrAux import ReqMgrAux
+from WMCore.Services.PhEDEx.PhEDEx import PhEDEx
+
+
+class MSCore(object):
+    """
+    This class provides core functionality for both
+    MSTransferor and MSMonitor classes.
+    """
+
+    def __init__(self, msConfig=None, logger=None):
```

Comment:
make `msConfig` mandatory and remove the check below

Reply:
done